Added Apsimx_Examples empty dialog to Crop Model sub-menu

MeSophie opened the pull request:
Fixes (partially) #7532 
This PR replaces #7616 following the comment of @lloyddewit 
@rdstern , @lloyddewit , @N-thony , I have added the empty dialog (Apsimx_Examples) now to `frmMain`
Kindly review and merge it so that I can proceed with the improvements

N-thony commented:
> Fixes (partially) #7532 This PR replaces #7616 following the comment of @lloyddewit @rdstern , @lloyddewit , @N-thony , I have added the empty dialog (Apsimx_Examples) now to `frmMain` Kindly review and merge it so that I can proceed with the improvements

@MeSophie could we have a quick call?

MeSophie commented:
@N-thony 

> > Fixes (partially) #7532 This PR replaces #7616 following the comment of @lloyddewit @rdstern , @lloyddewit , @N-thony , I have added the empty dialog (Apsimx_Examples) now to `frmMain` Kindly review and merge it so that I can proceed with the improvements
> 
> @MeSophie could we have a quick call?



> > Fixes (partially) #7532 This PR replaces #7616 following the comment of @lloyddewit @rdstern , @lloyddewit , @N-thony , I have added the empty dialog (Apsimx_Examples) now to `frmMain` Kindly review and merge it so that I can proceed with the improvements
> 
> @MeSophie could we have a quick call?

@N-thony  yes
